earth_enterprise/src/third_party/mod_wsgi/SConscript

```diff
@@ -125,6 +125,12 @@ mod_wsgi_env.ExecuteOnClean('rm -rf %s' % current_dir)
 
 if 'install' in COMMAND_LINE_TARGETS:
   server_root = mod_wsgi_env.installdirs['server_root']
+
+  # Configure mod_wsgi-ge.conf with the correct python version
+  mod_wsgi_conf = 'conf.d/mod_wsgi-ge.conf'
+  mod_wsgi_env.Command(mod_wsgi_conf, mod_wsgi_conf + '.in',
+      "sed 's/@PYTHON_VERSION@/%s/' < $SOURCE > $TARGET" % mod_wsgi_env['python_version'])
+
```

Comment:
I like adding of suffix ".in" for source file, but will prefer do not create output file in src directory.

The "install"-target is being used to copy all the files from bin_dir/.../install directories to specified install_dir to prepare them for deployment/packaging.
We need to move altering of mod_wsgi-ge.conf to the creating master installer target which prepares install-files in bin/../install dir (see line 103, command after mod_wsgi_target = '%s/.install' % current_dir)
In command on line 103, you may do something like:

mkdir -p %s/gehttpd/conf.d' % install_root_opt
install -m755 SOURCE_PATH TARGET_PATH
"sed -i -e 's/@PYTHON_VERSION@/%s/' TARGET_PATH" % mod_wsgi_env['python_version']

Source, target paths something like:
mod_wsgi_dir = Dir('#third_party/mod_wsgi').abspath
mod_wsgi_conf = 'conf.d/mod_wsgi-ge.conf'
SOURCE_PATH = path.join(mod_wsgi_dir, '%s.in' % mod_wsgi_conf)
TARGET_PATH = "%s/gehttpd/conf.d/%s' % (install_root_opt, mod_wsgi_conf)